lib/ansible/plugins/action/shell.py

```diff
@@ -13,8 +13,7 @@ class ActionModule(ActionBase):
     def run(self, tmp=None, task_vars=None):
         del tmp  # tmp no longer has any effect
 
-        # Shell module is implemented via command
```

Comment:
We don't need the `merge_hash` import (unrelated to this, just noticed it).

Reply:
`merge_hash` be gone! ;)